Dockerfile

```diff
@@ -15,13 +15,13 @@ FROM rust:1.76-alpine as backend
 WORKDIR /tmp
 RUN apk add --no-cache libc-dev openssl-dev alpine-sdk
 COPY ./packages/backend ./
-RUN cargo build --release
+RUN RUSTFLAGS="-Ctarget-feature=-crt-static" cargo build --release
```

Comment:
What is this for?

Reply:
The rust alpine image using libmusl can't build the binary without explicitly telling him to use shared libraries (by setting crt-static equals false). But I'm not a Rust expert, maybe you can find another way around.